packages/volto/src/components/manage/Toolbar/PersonalTools.jsx

```diff
@@ -50,7 +46,9 @@ const PersonalTools = (props) => {
   const token = useSelector((state) => state.userSession.token, shallowEqual);
   const user = useSelector((state) => state.users.user);
   const userId = token ? jwtDecode(token).sub : '';
-
+  const siteSetupAction = props.actions?.user?.find(
```

Comment:
Expecting `actions` to be passed as a new prop is a breaking change if Toolbar.jsx or PersonalTools.jsx was customized for a particular project.

It would be safer to get the actions from the redux store using useSelector.

```
const siteSetupAction = useSelector((state) => state.actions.actions)?.user?.find(
```